src/function.calls.spec.ts

```diff
@@ -155,4 +154,56 @@ describe("Function calls detector Agent Tests", () => {
     const findings: Finding[] = await handleTransaction(txEvent);
     expect(findings).toStrictEqual([findingGenerator({ to, from, input })]);
   });
+
+  it("Should be able to handle additional filter function passed inside functionCallGenerator", async () => {
+    const abiItem = {
+      name: "myMethod",
+      type: "function",
+      inputs: [
+        {
+          type: "uint256",
+          name: "myNumber",
+        },
+        {
+          type: "uint256",
+          name: "myNumber",
+        },
+      ],
+    } as AbiItem;
+
+    const filterFn = (value: TraceAction): boolean => {
+      const input = value.input;
+      const result = decodeFunctionCallParameters(["uint256", "uint256"], input);
```

Comment:
You can use this if you want to use the abiItem
```suggestion
      const result = decodeFunctionCallParameters(abiItem.inputs, input);
```

If you change the names of the parameters (currently both of them are `myNumber`) you will be able to access them by their name:
```typescript
const {name1, name2} = decodeFunctionCallParameters(...);
```
or
```typescript
const result = decodeFunctionCallParameters(...);
console.log(result.name1, result.name2);
```